datalad_next/patches/__init__.py

```diff
@@ -1,5 +1,6 @@
 from . import (
     annexrepo,
     create_sibling_ghlike,
+    push_to_export_remote,
```

Comment:
FTR: Import time increment seems to be 10ms. Not bad.

In comparison, the `create_sibling_ghlike` patch is 40ms, `annexrepo` is 95ms, `siblings` is 4ms.